spark/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM jupyter/pyspark-notebook:spark-3.1.1
+LABEL maintainer "peloton@lal.in2p3.fr"
+ENV REFRESHED_AT 2021-06-08
+
+# Add repo
+WORKDIR /home/jovyan/work
+ADD . .
+ENV PYTHONPATH=$PYTHONPATH:/home/jovyan/work/python:${SPARK_HOME}/python/test_coverage
```

Comment:
Is it really necessary to set a PYTHONPATH here? 

I would consider this bad practice

Reply:
`${SPARK_HOME}/python/test_coverage` needs to be appended to the `PYTHONPATH` to run tests and coverage with Spark (to gather the logs from all executors).

The `/home/jovyan/work/python` is not needed though (leftover from other project).

Reply:
See here for more info: https://github.com/apache/spark/blob/master/python/run-tests-with-coverage